ansible/roles/stack-sunbird/templates/sunbird_lms-service.env

```diff
@@ -90,6 +91,7 @@ sunbird_gzip_enable={{sunbird_gzip_enable}}
 sunbird_gzip_size_threshold={{sunbird_gzip_size_threshold | default(262144)}}
 sunbird_analytics_blob_account_name={{sunbird_private_storage_account_name}}
 sunbird_analytics_blob_account_key={{sunbird_private_storage_account_key}}
+sunbird_analytics_blob_account_endpoint={{sunbird_private_storage_account_endpoint}}
```

Comment:
@ddevadat what should be the default value for this variable sunbird_private_storage_account_endpoint ?
If this variable is required, we need to define in defaults.yml

Reply:
i didnt find any defaults for sunbird_private_storage_account_name , sunbird_private_storage_account_key in stack-sunbird role

Reply:
Lets initialise account_name, account_key and account_endpoint with below variables cloud_private_storage_accountname
cloud_private_storage_secret
cloud_private_storage_endpoint

Reply:
done